src/test/resources/META-INF/mods.toml

```diff
@@ -87,4 +87,6 @@ license="LGPL v2.1"
 [[mods]]
     modId="custom_tag_types_test"
 [[mods]]
-    modId="biome_loading_event_test"
\ No newline at end of file
+    modId="biome_loading_event_test"
+[[mods]]
+	modId="item_modifier_test"
```

Comment:
This is a tab, and should probably be four spaces to match the rest of the file.